index.d.ts

```diff
@@ -1584,6 +1584,29 @@ declare namespace Moleculer {
 			delimiter(value: string): void;
 		}
 	}
+
+	namespace Utils {
+		function isFunction(func: unknown): boolean;
```

Comment:
Return type here can be `func is Function` to match the other typeguard signatures.